assets/src/js/admin/onboarding-wizard/app/steps/introduction/index.js

```diff
@@ -21,7 +28,7 @@ const Introduction = () => {
 					<p>
 						{ __( 'You’re only minutes away from having a fully functional online donation platform on your website. We recommend using the setup wizard if this is your first time using Give.', 'give' ) }
 					</p>
-					<ContinueButton label={ __( 'Start Setup', 'give' ) } />
+					<ContinueButton clickCallback={ onStartSetup } label={ __( 'Start Setup', 'give' ) } />
```

Comment:
The callback prop should be passed as `clickCallback={() => onStartSetup() }`, currently this is firing the onStartSetup function as soon as the component renders, rather than actually on click.